lib/runner.js

```diff
@@ -44,7 +44,12 @@ class Runner {
     const cmd = this._templateCommand(this.config.build.before);
 
     this.services.logger.logRunner(this, `build [before] hook: ${cmd}`);
-    const exec = await this.container.exec(this._templateCommand(cmd));
+    const exec = await this.container.exec(
+      this._templateCommand(cmd),
+      {
```

Comment:
why not build this option into the constructor to avoid repeating it everywhere?